ui-ngx/src/app/shared/components/rule-chain/rule-chain-select.component.html

```diff
@@ -15,17 +15,13 @@
     limitations under the License.
 
 -->
-<mat-form-field class="tb-rule-select" subscriptSizing="dynamic">
+<mat-form-field class="tb-rule-select" subscriptSizing="dynamic" appearance="fill"
+                [ngClass]="{'cursor-pointer': !disabled}"
```

Comment:
Need to change angular binding [class](https://angular.io/guide/class-binding#binding-to-a-single-css-class).